applications/mne_analyze/libs/anShared/Management/analyzesettings.h

```diff
@@ -100,9 +100,68 @@ class ANSHAREDSHARED_EXPORT AnalyzeSettings : public QObject
     */
     virtual ~AnalyzeSettings();
 
+    QString bemName() const;                    /*!< Returns the current BEM file name. @return the BEM file name.*/
```

Comment:
since there are going to be many functions I would prefer keeping in analyze settings as well as in analyze data the docu style like for member variables.